hack/macros.py

```diff
@@ -1,6 +1,40 @@
 import os
+import semver
+import sys
+import traceback
+
+from github import Github
+
+# By default mkdocs swallows print() messages from macros
+def print_to_stdout(*a):
```

Comment:
usually *x should be *vargs and **kwargs for **

Reply:
done